dash/development/base_component.py

```diff
@@ -265,6 +323,137 @@ def __len__(self):
         return length
 
 
+def schema_is_nullable(type_object):
+    if type_object:
+        if type_object.get('name', None) == 'enum':
+            values = type_object['value']
+            for v in values:
+                value = v['value']
+                if value == 'null':
+                    return True
+        if type_object.get('name', None) == 'union':
```

Comment:
```suggestion
        if type_object.get('name') == 'union':
```

Default to get is already None.